features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/receipt/TimelineItemReadReceiptView.kt

```diff
@@ -81,7 +81,8 @@ fun TimelineItemReadReceiptView(
         }
     } else {
         when (state.sendState) {
-            LocalEventSendState.NotSentYet -> {
+            is LocalEventSendState.SendingFailed,
+            is LocalEventSendState.NotSentYet -> {
```

Comment:
I am not very happy to see `SendingFailed` rendered the same way as `NotSentYet`, but I understand that this is temporary, right?